### ISSUE TITLE
Font issues

### ISSUE THREAD

**Issue by bilekt3** [ionide/ionide-vscode-fsharp#241]
Maybe not directly ionide related but when I any ionide "project" I saw this. Html entity is rendered instead of ->.

I use:

Version 1.7.0-insider (1.7.0-insider) alongside with ionide 2.8.2

<img width="404" alt="font issues" src="https://cloud.githubusercontent.com/assets/381740/19766357/9d940d08-9c4d-11e6-9636-4b6044324f66.png">

Thanks for any kind of info

**Comment by Krzysztof-Cieslak** [comment]
Yes, it's known problem for Insiders version. 

1. Problem with escaping characters was noticed when we've first released CodeLens support
2. We've noticed that it's problem on VSCode side, which I've reported and PRed fix.
3. At the same time I've introduce temporary workaround on plugin side.
4. PR with my fix got merged to VSCode and is released in Insider version while Stable is not using it (it'll be released in usually monthly version) 
5. I decided to keep my plugin side workaround so plugin works well for Stable version until next Stable release. (which makes problem with Insiders)

Unfortunately, AFAIK there is nothing we can do with it for now (until next Stable release).
